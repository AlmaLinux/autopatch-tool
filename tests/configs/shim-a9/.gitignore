SOURCES/fbx64.efi
SOURCES/mmx64.efi
SOURCES/shimaa64.efi
SOURCES/shimx64.efi
