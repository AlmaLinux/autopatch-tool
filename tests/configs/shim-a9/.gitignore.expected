SOURCES/fbaa64.efi
SOURCES/fbx64.efi
SOURCES/mmaa64.efi
SOURCES/mmx64.efi
SOURCES/shimaa64.efi
SOURCES/shimx64.efi
